pwiz_tools/Skyline/Model/Lib/NistLibSpec.cs

```diff
@@ -937,7 +940,14 @@ private bool CreateCache(ILoadMonitor loader, IProgressStatus status, int percen
                             match = REGEX_PRECURSORMZ.Match(line);
                             if (match.Success)
                             {
-                                precursorMz = double.Parse(match.Groups[1].Value, CultureInfo.InvariantCulture);
+                                try
+                                {
+                                    precursorMz = double.Parse(match.Groups[1].Value, CultureInfo.InvariantCulture);

```

Comment:
Maybe if (!double.TryParse()) instead

Reply:
It's a nullable, so that would look something like
`
                                if (double.TryParse(match.Groups[1].Value, NumberStyles.Float, CultureInfo.InvariantCulture, out var parsedPrecursorMz))
                                {
                                    precursorMz = parsedPrecursorMz;
                                }
                                else
                                {
                                    ThrowIOException(lineCount, string.Format(Resources.NistLibraryBase_CreateCache_Could_not_read_the_precursor_m_z_value___0__, line) );
                                }
                                continue;
`